### ISSUE TITLE
QA Ph2_Accessibility: Attending - User cannot access map features using keyboard only to navigate

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#388]
Using the keyboard only to navigate the Attending page, the user is unable to access the following features on the map: pins, share, view larger map, zoom options.  
Navigation goes directly from the star icon to the maps Terms link. 

Steps: 
1. Go to https://phase2-dot-io-webapp-staging.appspot.com/io2016/
2. Select the Attending tab
3. Using the keyboard only, navigate to the map 
4. Note issue 

Browser: 
Chrome 48 Mac OS X 10.11 

![gio_388](https://cloud.githubusercontent.com/assets/7585604/13581882/ed78fe6c-e45d-11e5-9df7-da00f3e917ee.png)
